pkg/helm/step.go

```diff
@@ -2,7 +2,7 @@ package helm
 
 type Step struct {
 	Description string       `yaml:"description"`
-	Outputs     []HelmOutput `yaml:"outputs"`
+	Outputs     []HelmOutput `yaml:"outputs,omitempty"`
```

Comment:
This `omitempty` ensures that when we have tests that go from a go struct to yaml back to json to validate things (exhausting I know, but we have tests like that already), that we don't serialize an empty list of outputs and trigger more of the json validation.